website/src/releases/2022-12/custom-applications-multiple-permissions.mdx

```diff
@@ -0,0 +1,16 @@
+---
+date: 2022-12-15
+title: Custom Applications multiple permissions
+description: Introducing more granular permissions for Custom Applications.
+type: feature
+topics:
+  - Merchant Center
+  - Configuration
+  - CLI
+---
+
+The Application Kit packages have been released with a minor version `v21.21.0`.
+
+Custom Applications now enable optional configuration for defining more granular [OAuth Scopes and user permissions](/concepts/oauth-scopes-and-user-permissions#permission-groups) to cover more strict business requirements.
```

Comment:
Thanks for keeping it concise! 🤗

I would maybe elaborate a bit more and point to the updated parts of the documentation.

```suggestion
Custom Applications now allow to define more granular OAuth Scopes and user permissions to cover more strict business requirements. You can create [multiple permission groups](/concepts/oauth-scopes-and-user-permissions#permission-groups) and [assign different OAuth Scopes](/concepts/oauth-scopes-and-user-permissions#oauth-scopes) to them. These permission groups can be then [assigned to teams](/merchant-center/managing-custom-applications#assigning-team-permissions) in a more granular way.
```

Reply:
Thanks, updated 🙂